.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -45,12 +50,14 @@ Examples:
 - does this feature work in other server appplications (e.g. CMaNGOS, TrinityCore, etc...) ?
 -->
 
+
+
+
 ##### BRANCH(ES):
-<!-- Specify the branch(es) affected by this issue: master, 0.x, 1.x, or another branch. -->
+<!-- Specify the branch(es) affected by this issue: master. -->
```

Comment:
why changing this comment? If we are going to assume that everyone is always using `master` branch, then we can remove completely this section

Reply:
Afaik master is the only branch we use, I only keep it in case we'd do something different in the future

Reply:
yes, better change it else it's confusing